source_gen/test/generator_for_annotation_test.dart

```diff
@@ -142,47 +142,6 @@ $dartFormatWidth
     );
   });
 
-  test('applies to annotated directives', () async {
```

Comment:
What's the story here?

Reply:
Directives in the element model are no longer elements, so we cannot support annotations on them. Or at least the way it used to work. This so far did not cause any breakage in google3, but I have not yet migrated external packages, only a lot of internal generators.

Reply:
I asked in our [chat](https://chat.google.com/room/AAAA-IuSyjA/0e1s6XkTmCA/0e1s6XkTmCA?cls=10).